pages/index.vue

```diff
@@ -0,0 +1,76 @@
+<script>
+  export default {
+    data() {
+      return {
+        url: 'goose.png'
+      }
+    }
+  }
+</script>
+  
+<template>
+  
+    <div class="frontPage">
+      <div> <img :src="url"> </div>
```

Comment:
Take a look at the [Nuxt Asset](https://v3.nuxtjs.org/getting-started/assets/). Sorry, I didn't think about this last night!

Nuxt will automatically run some preprocessing on images imported from the `/assets` directory that will improve caching, etc..